src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -108,6 +88,28 @@ export interface MongoAdLibPiece {
   timelineObjectsString: string
 }
 
+export interface MongoStudio {
+  mappings: MongoLayerMappings
+  blueprintConfig: unknown
+}
+
+interface MongoLayerMappings {
+  [layerName: string]: MongoLayerMapping
+}
+
+export interface MongoShowStyle {
+  blueprintConfig: unknown
+}
+
+interface MongoLayerMapping {
+  // Which LookAhead "mode" we are in.
```

Comment:
This is more so i am sure which one is correct: 
But i assume this comment is supposed to say `lookAhead`.
Do we call it `lookAhead` in code when the database has it as `lookahead`?

Reply:
I'm up for unifying the spelling to be `lookahead` if there is a consensus to do it.
So people, feel free to weigh in :)

Reply:
It's been updated.
I might have missed some places since my IDE stops showing search results after the first 100+ results and one file had 350+ occurrences...